src/resources/js/bumpdown.js

```diff
@@ -32,8 +35,37 @@
 					data.$trigger.addClass( selectors.active.replace( '.', '' ) );
 					arrow = data.$trigger.position().left - ( 'block' === data.type ? data.$parent.offset().left : 0 );
 
-					$bumpdown.prepend( '<a class="tribe-bumpdown-close" title="Close"><i class="dashicons dashicons-no"></i></a>' );
-					$bumpdown.prepend( '<span class="tribe-bumpdown-arrow" style="left: ' + arrow + 'px;"></span>' );
+					var $content = $bumpdown.find( selectors.content );
+
+					if ( 'undefined' !== typeof width_rule && width_rule ) {
+						if ( 'all-triggers' === width_rule ) {
```

Comment:
Instead of two nested `if` blocks, could we bump this check to the one above, ie `( 'string' === typeof width_rule && 'all-triggers' === width_rule )`?